spec/system/referee_interface/referee_can_submit_a_reference_for_candidate_with_relationship_and_safeguarding_with_show_reference_confidentiality_status_feature_flag_inactive_spec.rb

```diff
@@ -0,0 +1,322 @@
+# This file can be deleted along with the show_reference_confidentiality_status feature flag. All other functionality is tested in spec/system/referee_interface/referee_can_submit_a_reference_for_candidate_with_relationship_and_safeguarding_spec.rb
+
+require 'rails_helper'
+
+RSpec.describe 'Referee can submit reference', :with_audited do
+  include CandidateHelper
+
+  around do |example|
+    old_references = CycleTimetable.apply_opens(ApplicationForm::OLD_REFERENCE_FLOW_CYCLE_YEAR)
```

Comment:
Why do we want to travel to 2022?

Reply:
This file will be deleted with the feature flag. It's a duplicate of `spec/system/referee_interface/referee_can_submit_a_reference_for_candidate_with_relationship_and_safeguarding_spec.rb`, the only difference is that the feature flag is switched off in this test. 

Because this test won't live for much longer i'd rather not spend any time on it and it should be the same as the other test.